components/script/dom/htmlimageelement.rs

```diff
@@ -782,6 +797,71 @@ impl LayoutHTMLImageElementHelpers for LayoutJS<HTMLImageElement> {
     }
 }
 
+pub fn parse_a_sizes_attribute(input: DOMString, width: Option<u32>) -> Result<Vec<Size>, String> {
```

Comment:
Since we only return Ok from this method (and not Err), there is no reason to return a Result value.